### DISCUSSION TITLE
大家来随便讨论呀👋

### DISCUSSION THREAD

**Discussion by Crescentm** [openwhu/OpenWHU#50] [General]
俺来举个栗子（随便讨论啥都行）

# 国内如何流畅访问github
> github在国内没办法访问的主要原因应该是dns污染
## 1. 改hosts
通过修改hosts文件指定访问github的服务器，从而避免打不开github。
网上修改hosts的教程有很多，可以自己搜索。
[如果你实在懒得搜索](https://yangshun.win/blogs/2b7abf4f/#%E6%9F%A5%E8%AF%A2%E7%BD%91%E5%9D%80-ip)， [那就点开我们之中一个](https://segmentfault.com/a/1190000039694021)
## 2. 梯子
最长期有效的方法还是梯子，而最稳定的梯子往往都需要付费，所以你懂得
## 3.各种镜像网站
[cnpmjs](https://github.com.cnpmjs.org/)

**Comment by GUZZ07** [comment]
21级小萌新打卡

**Reply by Crescentm** [reply]
表面萌新（实则大佬

**Reply by GUZZ07** [reply]
> 表面萌新（实则大佬

哇学长回我了，学长带带我！

**Reply by yyangzq** [reply]
打卡（

**Reply by Crescentm** [reply]
打卡圣地属于是了

**Reply by jackcheng794** [reply]
22级萌新打卡

**Reply by javasmoke** [reply]
24级萌新打卡

**Reply by miku3939-creator** [reply]
25级萌新打卡

**Reply by shenxianovo** [reply]
虽然现在发有点迟了...但是还是来补个23的（）
23级萌新打卡

**Comment by Zenitheee** [comment]
是不是可以考虑做做推广，我大三才知道这么个东西。。。

**Reply by Crescentm** [reply]
可能需要更多人加入项目维护来注入新鲜血液，仓库已经很久没活跃了

**Comment by liaosunny123** [comment]
https://github.com/WHUCodingAndOpen/OpenCourseResources

**Reply by Crescentm** [reply]
考虑archive这个项目，留链接指向[OpenCourseResources](https://github.com/WHUCodingAndOpen/OpenCourseResources)
